### DISCUSSION TITLE
Custom 404 page, jest and react-testing-library

### DISCUSSION THREAD

**Discussion by dandoniev** [vercel/next.js#38548] [Help]
Hey, guys. I am curious of inability to import or mock custom 404 page in unit tests. Once I try to do this I get something like: 
![image](https://user-images.githubusercontent.com/80757554/178454585-9026f816-a58b-4b3a-8f95-a54bd6ffd192.png)

Next: 12.1.6
Jest: 28.1.0
@testing-library/react: 13.2.0

Is there any way to mock this 404 page?

**Comment by icyJoseph** [comment]
Do you have a repository, reproduction demo to look at?

How are you running this test? Have you told jest how to resolve absolute paths?
